.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       UV_PYTHON: ${{ matrix.python-version }}
     strategy:
       matrix:
-        python-version: [ "3.10", "3.11" ]
+        python-version: [ "3.10", "3.12" ]
```

Comment:
3.11 is skipped. Are we using a min version, max version matrix?

Reply:
No just testing the min and max versions but we could test the 3 supported ones?